<launch>
	<arg name="affw" default="false" />

	<param name="joy/deadzone" value="0.33" />
	<node name="joy" pkg="joy" type="joy_node" output="screen" />
	
	<param name="vDef" value="1" />
	<param name="vMax" value="3" />
	<param name="dccDef" value="2" />
	<param name="dccMax" value="4" />
	<param name="accDef" value="2" />
	<param name="accMax" value="3" />
	<param name="rotateDef" value="3" />
	<param name="rotateMax" value="5" />
	<param name="rotateMin" value="0.5" />
	<param name="affw" value="$(arg affw)" />
	<node name="affw_joy" pkg="affw_joy" type="joy2vel" output="screen" />
</launch>